Code/Legacy/CryCommon/ILevelSystem.h

```diff
@@ -65,7 +65,11 @@ struct ILevelSystem
 
     virtual bool LoadLevel(const char* levelName) = 0;
     virtual void UnloadLevel() = 0;
-    virtual bool IsLevelLoaded() = 0;
+
+    // Deprecated. Use AzFramework::LevelSystemLifecycleInterface::Get()->IsLevelLoaded instead.
+    virtual bool IsLevelLoaded() const = 0;
+
+    // Deprecated. Use AzFramework::LevelSystemLifecycleInterface::Get()->GetCurrentLevelName instead.
```

Comment:
can you follow this: https://github.com/o3de/sig-core/blob/main/governance/API-Ref-Guidelines-Update.md#deprecations

Reply:
fixed! thank you